### PR TITLE
Removes region variable due to removal from AWS provider resource 

### PR DESCRIPTION
**Pull request by eemperor**
Terraform `terraform-aws-provider` removed `region` attribute from `aws_s3_bucket` in v3.0.0.

https://github.com/terraform-providers/terraform-provider-aws/issues/14441

### CODE REVIEW

File: .bumpversion.cfg [hunk 1]
```diff
@@ -1,7 +1,6 @@
 [bumpversion]
-current_version = 2.0.0
+current_version = 2.0.1
```
Comment (confusdcodr) [inline .bumpversion.cfg:2]: This is a backwards incompatible change so this should be a major bump

Comment (eemperor) [reply .bumpversion.cfg]: Bumped major version

**Comment by eemperor** [comment]
@plus3it/terraform Ready for another look.  I added the additional test and bumped the major version.

File: tests/create_bucket_v2/main.tf [hunk 2]
```diff
@@ -0,0 +1,27 @@
+provider "aws" {
+  alias   = "v2"
```
Comment (lorengordon) [inline tests/create_bucket_v2/main.tf:2]: remove this line, no need to set an alias.

Comment (eemperor) [reply tests/create_bucket_v2/main.tf]: Removed

**Pull request merged**